test/unit_tests/unittest_io_state_rosconversions.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#include <gtest/gtest.h>
+#include "psen_scan_v2/IOState.h"
+
+#include "psen_scan_v2/io_state_ros_conversion.h"
+
+using namespace psen_scan_v2;
+using namespace psen_scan_v2_standalone;
```

Comment:
```suggestion
namespace standalone = psen_scan_v2_standalone;
```
or remove it, because there is no benefit if you have to write it out below.